docs/caph/01-getting-started/03-preparation.md

```diff
@@ -34,11 +34,11 @@ It is a common practice to create a temporary, local bootstrap cluster, which is
 
 ### Choose one of the options below:
 
-#### 1. Existing Management Cluster.
+#### 1. Existing Management Cluster
 
-For production use, a “real” Kubernetes cluster should be used with appropriate backup and Disaster Recovery policies and procedures in place. The Kubernetes cluster must be at least a [supported version](https://github.com/syself/cluster-api-provider-hetzner/blob/main/README.md#%EF%B8%8F-compatibility-with-cluster-api-and-kubernetes-versions).
+For production use, a “real” Kubernetes cluster should be used with appropriate backup and Disaster Recovery policies and procedures in place. The Kubernetes cluster must be of a [supported version](/docs/caph/01-introduction).
```

Comment:
This link is wrong, it should be `/docs/caph/01-getting-started/01-introduction`